migration/main.py

```diff
@@ -34,12 +41,12 @@ def main(argv):
         csvfile = io.StringIO(lpass.get_lp_data())
 
     if is_migrating_folders:
-        print('Migrating folders...')
+        print('Migrating folders:')
         migrate_folders(csvfile)
```

Comment:
Wouldn't we want to skip migrating shared folders as well if the option is set?

Reply:
I'd agree with this. 

I took a stab at this here: https://github.com/1Password/solutions/commit/028d1df1e23fb1559dae9ce3e04cb2b3b5d02286

Did it on it's own branch because we've got two pending MRs and some bugs/regressions in both and I didn't want to muddy things up too much. If we want to cherrypick that commit over we can, but don't want to step on anyone's work in progress either.